docs/sources/write/shortcodes/index.md

```diff
@@ -547,6 +547,47 @@ Including the original image dimensions as the 'width' and 'height' properties i
 These values are _only_ used for determining the image aspect ratio and don't equate to the final displayed size.
 {{< /admonition >}}
 
+### Center an image example
+
+You can center an image using the `figure` shortcode by adding the following properties:
+
+- Add the `class="w-100p"` property.
+- Add the `link-class="w-fit mx-auto d-flex flex-direction-column"` property.
+- Add the `max-width="WIDTHpx"` property, replacing `WIDTH` with the value you want for your image. The value should be lower than the original image's width.
+- Optionally, you can add the `width` and `height` properties to help optimize the page. If you're using a raster image, such as a PNG or JPG file, you can set the properties with the original image values, without pixels or percentages. For example, for a 800x600 image, you can add `width="800" height="600"`. For a vector image, you have to calculate the values based on the width you want for the image with the formula: (new-width / full-width) * full-height = new-height. For a detailed explanation about this step, refer to the Center image properties section below.
```

Comment:
I think we should avoid hyphen separated words in a formula which could reasonably have `-` to mean subtraction.

We should also avoid "below" as it's not sympathetic to screen readers -- https://developers.google.com/style/word-list#below

nit: semantic line breaks and some paragraphs for readability.

```suggestion
- Optionally, you can add the `width` and `height` properties to help optimize the page.
  
  If you're using a raster image, such as a PNG or JPG file, you can set the properties with the original image values, without pixels or percentages.
  For example, for a 800x600 image, you can add `width="800" height="600"`.
  
  For a vector image, you have to calculate the values based on the width you want for the image with the formula: (new_width / full_width) * full_height = new_height.
  For a detailed explanation about this step, refer to the later **Center image properties** section.
```

Similar changes should be made to the explanation at the end of the collapsed section.